src/components/context_menu/context_menu_panel.tsx

```diff
@@ -161,17 +161,20 @@ export class EuiContextMenuPanel extends Component<Props, State> {
     if (this.props.items && this.props.items.length) {
       switch (event.key) {
         case cascadingMenuKeys.TAB:
-          // We need to sync up with the user if s/he is tabbing through the items.
-          const focusedItemIndex = this.state.menuItems.indexOf(
-            document.activeElement as HTMLElement
-          );
-
-          this.setState({
-            focusedItemIndex:
-              focusedItemIndex >= 0 &&
-              focusedItemIndex < this.state.menuItems.length
-                ? focusedItemIndex
-                : undefined,
+          setTimeout(() => {
```

Comment:
We generally prefer `requestAnimationFrame` over `setTimeout` when needing a quick "delay this for a frame" logic. Do you mind swapping?

```suggestion
          requestAnimationFrame(() => {
```

Reply:
Totes mcgotes!